global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.1.504"
+    "dotnet": "3.0.100-preview4-010963"
   },
```

Comment:
@jmarolf  I think this might be breaking the build. From the log file.
```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
  - The following frameworks were found:
      3.0.0-preview4-27525-12 at [D:\a\1\s\.dotnet\shared\Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.
```

Reply:
Let me revert this.